src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/private.html.twig

```diff
@@ -1,3 +1,27 @@
+{#**
```

Comment:
The headers are duplicated here (and some other twig files) The regexp probably didn't detect it because it had an extra space at the beginning of the comment `{# **` If you just remove this initial extra space it should be good